frame/staking/src/pallet/impls.rs

```diff
@@ -865,10 +855,10 @@ impl<T: Config> ElectionDataProvider<T::AccountId, BlockNumberFor<T>> for Pallet
 	fn voters(
 		maybe_max_len: Option<usize>,
 	) -> data_provider::Result<Vec<(T::AccountId, VoteWeight, Vec<T::AccountId>)>> {
-		debug_assert!(<Nominators<T>>::iter().count() as u32 == CounterForNominators::<T>::get());
-		debug_assert!(<Validators<T>>::iter().count() as u32 == CounterForValidators::<T>::get());
+		debug_assert!(<Nominators<T>>::iter().count() as u32 == Nominators::<T>::count());
```

Comment:
This is kinda pointless now to have. We already check these in `fn check_count` as well, which is called after every test.

Reply:
Okay that makes sense, I will have them removed.